internal/services/controller/controller.go

```diff
@@ -112,7 +114,12 @@ func New(
 		queue,
 		f.Core().V1().Events().Informer(),
 		eventType,
-		oomevents.Filter,
+		filters.Filters{
+			{
+				autoscalerevents.Filter,
+				oomevents.Filter,
+			},
```

Comment:
Lets make it consistent with the k8s world and make the first dimension OR, second dimension AND. Right now, the second dimension is OR, first is AND.